### PR TITLE
Edpub 1193 create new submission email template

### PR DESCRIPTION
**Pull request by ckd002**
# Description

Updating email template for creation of new submission

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1193

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Deploy relevant branch to SIT using Bamboo
3. Sign into the Dashboard on SIT.
4. Click the 'New Request' button.
5. Choose any daac and click the 'Select' button.
6. Verify that you receive an email that matches the expected design

---

## Known Issues
1. The name of the DAAC will return as undefined in the email. This is a know issue and should be addressed next sprint

### CODE REVIEW

**Review by DavisGauntUAH — APPROVED** [review]
Known daac bug will be fixed in ticket to implement email changes  Good to merge after sit merged to uat

File: src/nodejs/lambda-layers/message-util/src/templates/new-submission.js
Comment (camposeddie) [inline src/nodejs/lambda-layers/message-util/src/templates/new-submission.js:2]: Revisiting this PR now that it can be tested in SIT.

- The URLs should all be populated by an environment variable instead of being hardcoded to PROD.
- After `following temporary name:` the submission name + submission id is referenced; however, the dashboard page only references the submission name, so submission id can be removed from there.

File: src/nodejs/lambda-layers/message-util/src/templates/new-submission.js
Comment (camposeddie) [inline src/nodejs/lambda-layers/message-util/src/templates/new-submission.js:10]: Should use dynamic url

File: src/nodejs/lambda-layers/message-util/src/templates/new-submission.js
Comment (camposeddie) [inline src/nodejs/lambda-layers/message-util/src/templates/new-submission.js:20]: Should use dynamic url

File: src/nodejs/lambda-layers/message-util/src/templates/new-submission.js
Comment (camposeddie) [inline src/nodejs/lambda-layers/message-util/src/templates/new-submission.js:21]: Should use dynamic url

**Pull request merged**
